### DISCUSSION TITLE
Execution error in ReEDS_2021

### DISCUSSION THREAD

**Discussion by narata2019** [NREL/ReEDS-2.0#134] [Q&A: Execution]
Hi, ReEDS team:

I am very glad that the new version has been released. However, When I execute ReDES 2021 with the cases_test, there is execution error like this. I think I can not solve it by myself. Could you help me with it? Many thanks!

Siming

![error](https://user-images.githubusercontent.com/105667264/208464738-e777bcd4-9b05-4994-8b07-72ed8171ee9f.png)

**Comment by wesleyjcole** [comment]
Siming - it looks like you are running an intertemporal case here.  Were you intending to do that?  We test ReEDS for sequential solves, but have not run it intertemporally for some time, so it's quite possible that the intertemporal functionality is currently not working.

**Comment by narata2019** [comment]
Yes, this is the int case. The seq case is good under 2021 version with my computer, but the int case can not be executed successfully. If it means that the int solving mode would not be used in 2021 or later versions?

**Reply by wesleyjcole** [reply]
We haven't tested the intertemporal version for quite a while, so I'm not surprised that it isn't working.  You are welcome to try to reimplement it if you want.  I can't say how big of an effort it would be, but it should be functional with some reimplementation changes.

**Comment by narata2019** [comment]
ReEDS is a perfect and complex GEP model and it is too complex for us to understand. I don't think I have the ability to solve the problem(>_<). Previously I used 2020 version and I found that DR and Hydrogen updataed in 2021 version, which are very interesting topics. I am wondering if there are any public technical reports focusing on the effects of DR and H2 production on future power system?  Another question is about the model. I am curious what's the difference between flex load and DR?  Many thanks!

**Comment by wesleyjcole** [comment]
The most up-to-date publication covering the hydrogen representation is included in our report "Examining Supply-Side
Options to Achieve 100% Clean Electricity by 2035" (https://www.nrel.gov/docs/fy22osti/81644.pdf).  The most recent publication using the DR representation is at https://emp.lbl.gov/publications/assessing-interactive-impacts-energy.

**Reply by narata2019** [reply]
Many thanks!

**Reply by narata2019** [reply]
Hi, Dr. Wesley:

I test the ReEDS 2021 and ReEDS 2022 with ercot_test 2010-2030 scenario in seq solving mode with GSw_DR=1 to test the effects of demand response,  but both have same errors in the input function of  'all_year_load.py', as follows:
![1672235004003](https://user-images.githubusercontent.com/105667264/209821349-376db008-0546-4db3-a20c-9626f19029be.jpg)
The dataframe 'dr_data2' before the error line can be to_csv as this:
![1672235162602](https://user-images.githubusercontent.com/105667264/209821798-7e1c82cd-5b92-46b9-88a2-ed9a315fb635.jpg)
How can I solve the problem? Many thanks！

Siming

**Comment by wesleyjcole** [comment]
You have selected "none" for the drscen, so there is no DR data, which is what is throwing this error.  If you change drscen to "Baseline" then it should work fine (or at least my test run did).

**Reply by narata2019** [reply]
Yes, it works again! Thanks~ :）
